crates/re_log_types/src/component_types/color.rs

```diff
@@ -2,7 +2,8 @@ use arrow2_convert::{ArrowDeserialize, ArrowField, ArrowSerialize};
 
 use crate::msg_bundle::Component;
 
-/// An RGBA color tuple.
+/// An RGBA color tuple with unmultiplied/seprate alpha,
```

Comment:
```suggestion
/// An RGBA color tuple with unmultiplied/separate alpha,
```

Reply:
I trust you verified that that's how we use it everywhere? :)

Reply:
We don't support alpha anywhere 😆